### PR TITLE
Basic UI

### PR DESCRIPTION
**Pull request by ademibowale**
- I create a app.rb file that will serve as console app entry-point. It has a methods that do the following:
     - List all books.
     - List all people.
     - Create a person (teacher or student, not a plain Person).
     - Create a book.
     - Create a rental.
     - List all rentals for a given person id.
      
- In my main.rb i define the entry point, the method called main that is invoked at the end of my file. This method do the following:
     - Present the user with a list of options to perform.
     -  users can choose an option.

### CODE REVIEW

File: teacher.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization = 'Unknown')
+    super(age, name, parent_permission)
+    @specialization = specialization
+  end
```
Comment (VICTOR-CHUKWU) [inline teacher.rb]: - [ ] I am not able to create teachers, kindly update your initialize method to match the number of arguments passed when creating a teacher
image

File: student.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+require './person'
+
+class Student < Person
+  attr_reader :classroom
+
+  def initialize(classroom = 'Unknown')
+    super(age, name, parent_permission)
+    @classroom = classroom
+  end
```
Comment (VICTOR-CHUKWU) [inline student.rb]: - [ ] same as in teacher, kindly update the arguments in the student initialize to match the argument passed while creating student

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @ademibowale  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: student.rb (hunk 2 above)
Comment (ademibowale) [reply student.rb]: Thank you so much i have done this

File: teacher.rb (hunk 1 above)
Comment (ademibowale) [reply teacher.rb]: oh great job thank you so much i have fix it all kudos to you

File: main.rb [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  def options
+    print 'WELCOME TO THE SCHOOL LIBRARY APP'
+    while @output != 7
+      print "\nPlease choose an option by entering a number \n\n"
+      print "1 - List all books \n"
+      print "2 - List all people \n"
+      print "3 - Create a person (teacher or student) \n"
+      print "4 - Create a book \n"
+      print "5 - Create a rental \n"
+      print "6 - List all rentals for a person \n"
```
Comment (Ismailco) [inline main.rb:18]: - [ ] [OPTIONAL] Kindly consider printing the id of the person, so when you want to list the rentals the user can know the id of the person, as for now there is no way to get the person id to list the rentals

**Review by Ismailco — APPROVED** [review]
### Hi @ademibowale, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**